Dockerfile.centos.ppc64le

```diff
@@ -0,0 +1,42 @@
+FROM debian:buster-slim as qemu
+
+RUN apt-get update && apt-get install --no-install-recommends -y qemu-user-static
+
+FROM nvidia/cuda-ppc64le:8.0-devel-centos7 as build
+
+COPY --from=qemu /usr/bin/qemu-ppc64le-static /usr/bin/qemu-ppc64le-static
+
+RUN yum install -y \
+        gcc-c++ \
+        ca-certificates \
+        wget \
+        cuda-cudart-dev-8-0 \
+        cuda-misc-headers-8-0 \
+        cuda-nvml-dev-8-0 && \
+    rm -rf /var/cache/yum/*
+
+ENV GOLANG_VERSION 1.10
+RUN wget -nv -O - https://storage.googleapis.com/golang/go${GOLANG_VERSION}.linux-ppc64le.tar.gz \
+    | tar -C /usr/local -xz
+ENV GOPATH /go
+ENV PATH $GOPATH/bin:/usr/local/go/bin:$PATH
+
+ENV CGO_CFLAGS "-I /usr/local/cuda-8.0/include"
+ENV CGO_LDFLAGS "-L /usr/local/cuda-8.0/lib64"
+ENV PATH=$PATH:/usr/local/nvidia/bin:/usr/local/cuda/bin
+
+WORKDIR /go/src/nvidia-device-plugin
+COPY . .
+
+RUN export CGO_LDFLAGS_ALLOW='-Wl,--unresolved-symbols=ignore-in-object-files' && \
+    go install -ldflags="-s -w" -v nvidia-device-plugin
+
+
+FROM ppc64le/centos:7
```

Comment:
@clnperez is that the right place now? no more `ibmcom/centos`?

Reply:
it looks like we've finally got an official centos image, so this is what we should use. if you weren't using qemu, you could use centos:7, since it's a multi-arch image (confirmed with `docker manifest inspect -v centos:7`)